Source/WebKit/WebProcess/GPU/graphics/RemoteGraphicsContextGLProxy.cpp

```diff
@@ -247,7 +247,7 @@ RefPtr<Image> RemoteGraphicsContextGLProxy::videoFrameToImage(WebCore::VideoFram
     RefPtr<NativeImage> nativeImage;
 #if PLATFORM(COCOA)
     callOnMainRunLoopAndWait([&] {
-        nativeImage = m_videoFrameObjectHeapProxy->getNativeImage(frame);
+        nativeImage = RefPtr { m_videoFrameObjectHeapProxy }->getNativeImage(frame);
```

Comment:
Why not add `protectedVideoFrameObjectHeapProxy()`

Reply:
Same as above.